hasher-matcher-actioner/.devcontainer/devcontainer.json

```diff
@@ -24,6 +24,7 @@
 	"mounts": [
 		"source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind",
 		"source=${localEnv:HOME}${localEnv:USERPROFILE}/.aws,target=/var/run/aws-config,type=bind,consistency=cached",
+		"source=${localEnv:HOME}${localEnv:USERPROFILE}/.hma-cmdhist,target=/commandhistory,type=bind",
```

Comment:
@schatten FYI: building the docker image ended up breaking for me until I ran `mkdir .hma-cmdhist` in my home dir locally. (could be a local issue due to my set up but it might be worth calling out.)